pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.yapf]
```

Comment:
what is this file for? @n-poulsen

Reply:
It's a [configuration file](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/) used by the linters we use (e.g. [Black](https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#configuration-via-a-file), isort) so we all have that we all have a common configuration

It was added by @jeylau, and I think we should actually update it now - it configures YAPF styling but we use Black.